test/utils/utils.go

```diff
@@ -58,7 +58,7 @@ const (
 )
 
 func warnError(err error) {
-	fmt.Fprintf(GinkgoWriter, "warning: %v\n", err)
+	GinkgoLogr.Error(err, "warning error occurred")
```

Comment:
Should we remove `GinkgoWriter` initialization from `internal/controller/suite_test.go:57`?

Reply:
No because when you set the logging to very verbose then it prints the output from the operator. And that line allows it to show up. So we can leave that.